types/react-native-autocomplete-input/react-native-autocomplete-input-tests.tsx

```diff
@@ -1,34 +1,58 @@
-import Autocomplete from "react-native-autocomplete-input";
```

Comment:
Apologies for the diff noise. Running prettier replaced double-quotes with single-quotes and updated indents to be 4 spaces instead of 2.